### DISCUSSION TITLE
Docker images for all the services

### DISCUSSION THREAD

**Discussion by coolbung** [Sunbird-Ed/Community#241] [Installation] [answered]
Currently the Sunbird installation needs adopters to build each service. Often, the builds run into errors due to software version mismatch (Ubuntu, Python, Node, Maven etc versions)

Will sunbird provide ready docker images that can be used, thus improving the installation experience for adopters ?

**Comment by Krishnaj20** [comment]
Tagging @gandham-santhosh

**Reply by gandham-santhosh** [reply]
@coolbung @Krishnaj20 This item is already there in our roadmap and we will be addressing this soon 
cc: @keshavprasadms

**Reply by chitranshu-keshav** [reply]
@gandham-santhosh Can you please update this thread with the JIRA link of the issue related to this query in your Roadmap

cc: @Krishnaj20

**Comment by Krishnaj20** [comment, marked as answer]
@coolbung - this is now addressed in https://github.com/orgs/Sunbird-Ed/discussions/308. Please follow this thread. Marking this thread as answered.
